typescript-typing-tutor/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Typing Tutor</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nerko+One&display=swap"
+      rel="stylesheet" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Typing Tutor!</h1>
+      <h2>
+        <span class="letter currentLetter">t</span>
+        <span class="letter">h</span>
+        <span class="letter">i</span>
+        <span class="letter">s</span>
+        <span class="letter space">_</span>
+        <span class="letter">i</span>
+        <span class="letter">s</span>
+        <span class="letter space">_</span>
+        <span class="letter">v</span>
+        <span class="letter">e</span>
+        <span class="letter">r</span>
+        <span class="letter">y</span>
+        <span class="letter space">_</span>
+        <span class="letter">t</span>
+        <span class="letter">e</span>
+        <span class="letter">d</span>
+        <span class="letter">i</span>
+        <span class="letter">o</span>
+        <span class="letter">u</span>
+        <span class="letter">s</span>
+      </h2>
+      <h2 class="accuracy"></h2>
```

Comment:
Rather than creating this here and giving it textContent, I would have probably created the element in TypeScript and appended it when needed. You generally want to avoid having empty content in your HTML because screen readers will read it out and it can be confusing